widget/public/brandpay.js

```diff
@@ -50,12 +40,27 @@ async function initialize() {
   });
 }
 
-// 결제하기
+document
+  .querySelector('#payment-form')
+  .addEventListener('submit', handleSubmit);
+
+document.querySelector('#button').addEventListener('click', updateAmount);
+
+// 결제 금액 업데이트
+async function updateAmount(e) {
+  e.preventDefault();
+  paymentMethodsWidget.updateAmount(50000);
+}
+
+// 결제 하기
 async function handleSubmit(e) {
   e.preventDefault();
 
+  // 위젯 결제 정보
+  const widgetPaymentParams = paymentMethodsWidget.getPaymentParams();
+
   await brandpay.requestPayment({
```

Comment:
```suggestion
  await brandpay.requestPayment({
    orderId: 'ORDER_ID', // 주문에 대한 고유한 ID 값
    orderName: '생수 외 1건', // 결제에 대한 주문명
    successUrl: window.location.origin + '/success',
    failUrl: window.location.origin + '/fail',
    ...widgetPaymentParams,
```

갑자기 든 생각인데 이렇게 작성하면 더 보기 편할것 같지 않나요?!

Reply:
넵 좋아요~

Reply:
14, 68, 69 라인 코드 지워도 될 것 같아요 ㅋㅋㅋ 🙇